charts/karpenter-crd/templates/karpenter.k8s.aws_ec2nodeclasses.yaml

```diff
@@ -4,6 +4,9 @@ kind: CustomResourceDefinition
 metadata:
```

Comment:
Why uninstall the `karpenter-crd` chart if you don't want the CRDs to be removed? Is there another reason that you would be removing the chart?

Reply:
it is more if it gets uninstalled by accident, like wehn deploying the helm chart with argocd and the argocd deployment gets deleted

Reply:
i see othet charts like cert-manager also having an option to set this helm keep annotation to the crds so